docs/Usage/Cells.md

```diff
@@ -0,0 +1,100 @@
+* #### Markdown Cells
+These cells contain text formatted using [common markdown](https://www.markdownguide.org/cheat-sheet/). All kinds of formatting features are available like making text bold and italic, displaying ordered or unordered list, rendering tabular contents etc. Markdown cells are especially useful to provide documentation to the computational process.
+* #### Raw Cells
+Contents in raw cells are not evaluated by the kernel. When passed, they will be rendered as desired.
+
+
+## How to create Cells in Hydrogen
+Hydrogen supports a multitude of ways to make cells. Cells are  defined using "Cell markers".
+
+### Cell Markers
+
+A cell marker is a special inline comment that defines the lines that begin each cell. The inline comment must start with the comment character registered in the language's grammar file.
+
+- Python: `#`
+- R: `#`
+- Julia: `#`
+- Javascript: `//`
+- C++: `//`
+- Stata: `//`
+- Matlab: `%`
+
+
+### Example Definitions
+
+#### Code Cells
+
+Code cells are defined, by adding the following markers, after the comment character specefic to the language you are using.
+
+##### Markers:
+- `%% codecell`
+- `<codecell>`
+- `In[]` (you can have any number of charaters inside the brackets (empty also works))
+
+For example, the following text corresponds to three code cells in a Python file
+```py
+# %% codecell
+print('Hello, world!')
+# <codecell>
+print('foo')
+# In[1]
+print('bar')
+```
+The equivalent code in JavaScript will be
+```js
+// %% codecell
+print('Hello, world!');
+// <codecell>
+print('foo');
+// In[1]
+print('bar');
+```
+
+These are the same cell markers used by [**"Hydrogen: Run Cell"**](GettingStarted.md#hydrogen-run-cell).
+
+#### Markdown Cells
+
+You can also create cells with [Markdown](https://www.markdownguide.org/cheat-sheet/)-formatted text. Just as for code cells, there are a range of allowed markers to delimit the start of a markdown cell:
+
+- `%% md`
+- `%% [md]`
+- `%% markdown`
+- `%% [markdown]`
+- `<md>`
+- `<markdown>`
+
+Each line of text in the cell's body should be prefixed by the language's comment character. Any common leading whitespace in the cell will be stripped, so the body text can include a space after the comment symbol without having leading whitespace in the output and with indented lists keeping their structure.
+
+For example, here are two markdown cells in a Python file
+```
+# %% [markdown]
+# _italic text_
+# **bold text**
+# <markdown>
+# - Indented
+#     - Lists
+```
+
+the two cells would be exported without any common leading whitespace as
+```md
+_italic text_
+**bold text**
+```
+and
+```md
+- Indented
+    - Lists
+```
+
+#### Raw Cells
+
+Hydrogen doesn't support the export of [raw cells](https://nbformat.readthedocs.io/en/latest/format_description.html#raw-nbconvert-cells) at this time.
```

Comment:
why are you mentioning export here, we dont even accept their creations in hydrogen. Just leave as currently unsupported

Reply:
@wadethestealth 
> why are you mentioning export here, we dont even accept their creations in hydrogen. Just leave as currently unsupported

It was in the original documentation so I left it as it is.
I'm removing it for now since we already mentioned that they are not supported at the top and there's no need to add it in Examples per say.
Do you want me to add a relevant link to the prementioned section?